### ISSUE TITLE
a11y: Video widget accessibility

### ISSUE THREAD

**Issue by ramiy** [pojome/elementor#3265]
## Description

The video widget need some accessibility love!

1. **Name** - The iframe should have an accessibility name (https://fae.disability.illinois.edu/rulesets/FRAME_2/), either `title` or `aria-label` tags.

2. **Action** - The play icon should have some text saying "Play video" or some thing like that. Currently, regular visitors **see** the play button and understand that clicking the icon actives the video, but blind people **don't see** the icon. We need to add an assistive text for them.

3. **Markup** - The play icon wrapper is a `<div>`, it should be a `<button>`.

![video-a11y](https://user-images.githubusercontent.com/576623/35197381-2c328a24-fee7-11e7-96c7-6ddf18ae6e20.png)
